### PR TITLE
fix virtualenv and pip version

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
attempt to fix pip version when in virtualenv

### CODE REVIEW

**Comment by joshmoore** [comment]
Does this require a re-generation of any docs?

**Comment by aleksandra-tarkowska** [comment]
already done in https://github.com/openmicroscopy/ome-documentation/pull/1521

**Comment by joshmoore** [comment]
What's the status of m4 _without_ this PR?

File: ansible/templates/centos7/deps.sh.j2 [hunk 1]
```diff
@@ -13,7 +13,11 @@ yum -y install git
...
 yum -y install \
+    python-devel \
     python-pip \
-    python-virtualenv \
     libjpeg-devel \
     libtiff libtiff-devel
+
+# install the latest version
+pip install --upgrade pip
+pip install virtualenv
```
Comment (rleigh-codelibre) [inline ansible/templates/centos7/deps.sh.j2]: Missing newline at EOF

File: ansible/templates/common/arg_vars.j2 [hunk 2]
```diff
@@ -6,4 +16,5 @@ WEBSESSION=${WEBSESSION:-{{ web_session }}}
 
 WEBPREFIX=${WEBPREFIX:-{{ default.web_prefix }}}
 WEBPORT=${WEBPORT:-{{ default.web_port }}}
-WEBSERVER_NAME=${WEBSERVER_NAME:-{{ default.web_server_name }}}
\ No newline at end of file
+WEBSERVER_NAME=${WEBSERVER_NAME:-{{ default.web_server_name }}}
+{% endif %}
```
Comment (rleigh-codelibre) [inline ansible/templates/common/arg_vars.j2]: Missing newline at EOF

File: ansible/templates/common/create_user.sh.j2 [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {% if doc %}
-Create local user {{ omero_user }}, homedir `{{ omero_user_home_dir }}`::
+Create local user {{ omero_user }}, homedir `{{ omero_user_home_dir }}` (run as root)::
```
Comment (rleigh-codelibre) [inline ansible/templates/common/create_user.sh.j2]: Note here (and any other place using paths) that the backticks are not appropriate sphinx markup.  Use them doubled up to be verbatim, or with `:file:` to specifically mark up a path.  Single backticks mean something completely different.

**Comment by rleigh-codelibre** [comment]
In general this all looks fine.  Some minor markup and EOF newline issues noted.

**Comment by joshmoore** [comment]
Thanks the feedback, @rleigh-codelibre. I'll leave this open for @aleksandra-tarkowska to update and re-generate, but merge https://github.com/openmicroscopy/ome-documentation/pull/1521 as ok state of m4.

**Comment by joshmoore** [comment]
What's the next step here?

**Comment by aleksandra-tarkowska** [comment]
there can be more work on style, but at the moment is enough to fully test it with autogen. https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs-autogen/ was fixed and now can be staged.

**Comment by aleksandra-tarkowska** [comment]
@joshmoore travis is green this is good to merge then

**Comment by jburel** [comment]
* I will use ``latest`` in the README for example instead of ``OMERO-DEV-breaking-build``
 * Feedback we had while creating the omero-install was to make it clear which user is running the command. I will be preferable to have **Run as root** ... instead of  ``(run as root)`` e.g. ``Install other dependences (run as root)``

**Comment by aleksandra-tarkowska** [comment]
@jburel README updated. Would you agree if we go through templates in another PR? as I said it will be more work done on https://github.com/ome/omeroweb-install/pull/3#issuecomment-249249747. At the moment priority was to get our build system to consume this repo. I will work with @hflynn on the style.

**Comment by jburel** [comment]
@aleksandra-tarkowska: understood for the formatting
could you also remove ``breaking`` for
``OS=centos7 ICEVER=3.6 OMEROVER=OMERO-DEV-breaking-build WEBPREFIX=/omero TRAVIS=False .travis/before_install.sh
``
(3 similar commands)

**Comment by aleksandra-tarkowska** [comment]
@jburel https://github.com/aleksandra-tarkowska/omeroweb-install/blob/911c27b8b52bd1b7530d2e147a9afb9a1ede0ffe/.travis.yml are all changed, see https://travis-ci.org/ome/omeroweb-install/builds/163064483

**Comment by jburel** [comment]
not in the README

**Comment by aleksandra-tarkowska** [comment]
done

**Comment by jburel** [comment]
Thanks

**Pull request merged**